server/data/restClient.ts

```diff
@@ -92,51 +93,52 @@ export default class RestClient {
     return ''
   }
 
-  async get({
+  async get<Response = unknown>({
```

Comment:
Updated this inline with the TS template. Instead of returning `Promise<unknown>` client calls can now return the specified response type.